bool detected
float64 max_area
int32 num_boxes
float64 confidence
float64 percentage_cover